tests/integration/test_tensorzero_agent_functions.py

```
@@ -23,6 +23,23 @@
 TENSORZERO_URL = os.getenv("TENSORZERO_BASE_URL", "http://localhost:3030")
 
 
+def is_tensorzero_available() -> bool:
+    """Check if TensorZero gateway is reachable."""
+    try:
+        with httpx.Client(timeout=5.0) as client:
+            response = client.get(f"{TENSORZERO_URL}/health")
+            return response.status_code == 200
+    except (httpx.ConnectError, httpx.TimeoutException, Exception):
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)